R/create.violinplot.R

```diff
@@ -25,7 +25,8 @@ create.violinplot <- function(
 	xleft.rectangle = NULL, ybottom.rectangle = NULL, xright.rectangle = NULL, ytop.rectangle = NULL,
 	col.rectangle = 'transparent', alpha.rectangle = 1, height = 6, width = 6, resolution = 1600,
 	size.units = 'in', enable.warnings = FALSE, description = 'Created with BoutrosLab.plotting.general',
-	style = 'BoutrosLab', preload.default = 'custom', use.legacy.settings = FALSE, disable.factor.sorting = FALSE
+	style = 'BoutrosLab', preload.default = 'custom', use.legacy.settings = FALSE, disable.factor.sorting = FALSE,
+	strip.col = 'white', strip.cex = 1,	strip.fontface = 'bold', layout = NULL
```

Comment:
@dan-knight : the new parameters are `strip.col, strip.cex, strip.fontface, layout` - all used in `create.scatterplot` and `create.boxplot`